dojo/tools/checkmarx/parser.py

```diff
@@ -79,8 +72,10 @@ def get_findings(self, filename, test):
 
                 if language is not None:
                     findingdetail = "{}**Language:** {}\n".format(findingdetail, language)
-                    if language not in self.language_list:
-                        self.language_list.append(language)
+                    if language not in language_list:
+                        language_list[language] = 1
+                    else:
+                        language_list[language] = language_list[language] + 1
```

Comment:
How much sense makes it to count the number of files for a language? You only get the number of files with vulnerabilities, right?

Reply:
Yes. but better than what we have for now: it was only the language so the graphs were bugged.
My opinion is we don't have enough data so we should completely get ride of this.

Reply:
I wouldn't miss it.

Reply:
Let's do that in another PR. for now it keeps things as before, I will ask users and remove them if not required anymore